### PR TITLE
Change to use the same subcommand syntax as subkey

### PR DESCRIPTION
**Pull request by NukeManDan**
For consistency between the two tools, the subcommand for `<node bin> key inspect-key` is changed to match `subkey` syntax ( `subkey inspect`)

This _may_ need modification of downstream dependencies in Polkadot and other code bases. I do not see this is the case, bu would be good to have a double-check on that and a companion PR made if so.

Alternatively to this PR, changing the subkey `inspect` command to `inspect-key` would have the same (but more verbose) effect.

Compliments/Related to #8674

### CODE REVIEW

**Review by tomusdrw — APPROVED** [review]
makes sense to me!

File: client/cli/src/commands/inspect_key.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ use structopt::StructOpt;
 	name = "inspect-key",
 	about = "Gets a public key and a SS58 address from the provided Secret URI"
 )]
-pub struct InspectKeyCmd {
+pub struct InspectCmd {
```
Comment (bkchr) [inline client/cli/src/commands/inspect_key.rs:31]: Why is that renamed? please rename it back to the old name.

File: client/cli/src/commands/inspect_key.rs [hunk 2]
```diff
@@ -28,7 +28,7 @@ use structopt::StructOpt;
 	name = "inspect-key",
```
Comment (tomusdrw) [inline client/cli/src/commands/inspect_key.rs:28]: Just noticed that this should have been changed?

File: client/cli/src/commands/inspect_key.rs (hunk 1 above)
Comment (NukeManDan) [reply client/cli/src/commands/inspect_key.rs:31]: It is changed to match the name of the command itself, I don't _think_ this is required for the CLI package we are using, is it? 
if not, of course, I can change back

File: client/cli/src/commands/inspect_key.rs (hunk 2 above)
Comment (NukeManDan) [reply client/cli/src/commands/inspect_key.rs:28]:
```suggestion
	name = "inspect",
```

File: client/cli/src/commands/inspect_key.rs (hunk 1 above)
Comment (bkchr) [reply client/cli/src/commands/inspect_key.rs:31]: It is not required to match.

Comment (NukeManDan) [reply client/cli/src/commands/inspect_key.rs]: reverted

**Comment by NukeManDan** [comment]
@bkchr I compiled and this is working with the `substrate key inspect` command as desired, with the struct reverted as requested.

**Review by bkchr — APPROVED** [review]
